bluzelled/src/main.cpp

```diff
@@ -1,54 +1,39 @@
 #include "Node.h"
 #include "NodeUtilities.h"
 #include "web_sockets/WebSocketServer.h"
-#include "web_sockets/WebSocket.h"
-#include "web_sockets/Listener.h"
-#include "web_sockets/Session.h"
-#include "server/server.hpp"
+#include "http/server/ServiceManager.h"
 #include "NodeManager.h"
 
 #include <boost/exception/all.hpp>
-#include <boost/beast/core.hpp>
 #include <boost/beast/http.hpp>
-#include <boost/beast/version.hpp>
-#include <boost/asio/ip/tcp.hpp>
-#include <boost/asio/strand.hpp>
-#include <boost/config.hpp>
-#include <iostream>
-#include <string>
 
 static unsigned long s_max_nodes = 25;
 static unsigned long s_min_nodes = 5;
 static Nodes s_nodes;
+
 static boost::mutex *s_mutex = nullptr;
 
 // List of nodes that were died and need to be reported to GUI.
 static Nodes s_removed_nodes;
 static boost::mutex s_removed_nodes_mutex;
-Nodes* get_removed_nodes() { return &s_removed_nodes; }
-boost::mutex& get_removed_nodes_mutex() { return s_removed_nodes_mutex; }
```

Comment:
s_removed_nodes, s_removed_nodes_mutex and corresponding methods are not used anymore. Seems like they are remnants of some merge.